vignettes/reportOutputStyle.css

```diff
@@ -0,0 +1,3 @@
+<style>
+div.reportOutput { background-color:#e6f0ff; border-radius: 5px; padding: 20px; border-style: solid none solid solid;}
```

Comment:
Style for report outputs: first draft. Report outputs look then like below. We can adjust it to whatever is better looking
![grafik](https://user-images.githubusercontent.com/25061876/97917850-6cd11700-1d55-11eb-9346-e7d838d7e564.png)

Reply:
That's awesome. 
This is centralized as a .css file + a style class to initialize and call in the vignette using the html tag anytime we need it.